### ISSUE TITLE
Ethereum Push Notifications Support on Polygon

### ISSUE THREAD

**Issue by crisgarner** [cryptexfinance/community-coordination#11]

**Comment by lenhartjames** [comment]
Just wanted to ask. Would this cover a use-case for receiving alerts about your vault ratio?

**Comment by crisgarner** [comment]
> Just wanted to ask. Would this cover a use-case for receiving alerts about your vault ratio?

Yes! that's actually the goal to send notifications to users related to their position on their vault.
